lib/testing/src/lib/process-services-cloud/pages/task-details-cloud-component.page.ts

```diff
@@ -15,8 +15,9 @@
  * limitations under the License.
  */
 
-import { Util } from '../../../util/util';
+// import { Util } from '../../../util/util';
 import { element, by } from 'protractor';
+import { BrowserVisibility } from '../../core/browser-visibility';
 
 export class TaskDetailsCloudComponent {
```

Comment:
The taskDetailsCloud is not a cloud component but a custom ADF one it is wrapping the <adf-cloud-task-header
So, on the cloud we should have a TaskHeaderCloudPage used by a custom page taskDetailsCloudDemoshell (it will not be part of the adf-testing)